File: Java/Exception/Pow.java
import java.lang.*;

class Calculadora {
    public static int power(int n, int p) {
    	if (n < 0 || p < 0) throw new IllegalArgumentException("n ou p");
        return (int)Math.pow(n, p);
    }
    public static void main(String[] args) {
        System.out.println(power(10, 2));
    }
}

//https://pt.stackoverflow.com/q/118292/101

File: C/Math/Pow.c
#include <stdio.h>

double power(double n, int p) {
    double pow = 1;
    for (int i = 1; i <= p; i++) pow *= n;
    return pow;
}

int main() {
    printf("%.0f\n", power(5, 2));
    printf("Enter n and p (n^p): ");
    double n;
    int p;
    scanf("%lf", &n);
    scanf("%d", &p);
    printf("The pow is: %.0f\n", power(n, p));
}

//https://pt.stackoverflow.com/q/296497/101
